ai-decision-services/ai-ad-dpp-engine/setup-function/setup-function.md

```diff
@@ -0,0 +1,44 @@
+Lab 6: Set up OCI Function
```

Comment:
==> Setup OCI Function

(Although, I would prefer using 'Configure' instead of 'Setup')